modules/core/arrow-test/src/main/kotlin/arrow/test/laws/TraverseLaws.kt

```diff
@@ -52,26 +55,32 @@ object TraverseLaws {
       )
   */
 
-  fun <F> laws(TF: Traverse<F>, FF: Functor<F>, cf: (Int) -> Kind<F, Int>, EQ: Eq<Kind<F, Int>>): List<Law> =
-    FoldableLaws.laws(TF, cf, Eq.any()) + FunctorLaws.laws(FF, cf, EQ) + listOf(
-      Law("Traverse Laws: Identity") { TF.identityTraverse(FF, cf, EQ) },
-      Law("Traverse Laws: Sequential composition") { TF.sequentialComposition(cf, EQ) },
-      Law("Traverse Laws: Parallel composition") { TF.parallelComposition(cf, EQ) },
-      Law("Traverse Laws: FoldMap derived") { TF.foldMapDerived(cf) }
+  fun <F> laws(TF: Traverse<F>, FF: Functor<F>, GENK: GenK<F>, EQK: EqK<F>): List<Law> =
```

Comment:
`Traverse<F>` implies `Functor<F>` afaik, so maybe we should remove it.

Reply:
done